### ISSUE TITLE
Pawdraw SVGS are being blocked by content security policy

### ISSUE THREAD

**Issue by lramos15** [microsoft/vscode-extension-samples#394]
Steps to reproduce
1. Run the custom editor sample
2. Open the pawDraw file
3. 🐛 The paws are missing

In the dev console it says `Failed to load resource: net::ERR_BLOCKED_BY_CLIENT` for both paw-color.svg and paw-outline.svg

**Comment by mjbvz** [comment]
Duplicate of https://github.com/microsoft/vscode/issues/118181
